backward_updates.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# Backward updates mechanism
+
+set -e
+
+source /usr/local/etc/library.sh
+
+# Get the array of updates dir
+# The files in updates dir are sorted by tag number
+# Files names follow the syntax of a tag: x.y.z.sh
+i=0
+while read line ; do
+  updates_list[ $i ]="$line"
```

Comment:
```suggestion
  updates_list +=("$line")
```
you don't need the `i` variable